File: dockerfile.py
# Imagem base leve
FROM alpine:3.18

# Diretório de trabalho
WORKDIR /app

# Criar arquivo de build simulado
RUN echo "Aplicação simulada - build em $(date)" > build.txt

# Expor porta apenas para fins de teste
EXPOSE 3000

# Comando para manter o container rodando
CMD ["sh", "-c", "while true; do sleep 3600; done"]

File: dockerfile.js
# Imagem base Node.js
FROM node:18-alpine

# Diretório de trabalho
WORKDIR /app

# Copiar arquivos de dependências
COPY package*.json ./

# Instalar dependências
RUN npm install

# Copiar o restante do código
COPY . .

# Expor porta usada pela aplicação
EXPOSE 3000

# Comando para rodar a aplicação
CMD ["node", "src/app.js"]
